Marlin/Configuration.h

```diff
@@ -433,6 +433,64 @@ const bool Z_MIN_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the l
 //#define Z_PROBE_SLED
 //#define SLED_DOCKING_OFFSET 5 // The extra distance the X axis must travel to pickup the sled. 0 should be fine but you can push it further if you'd like.
 
+// With this option servos are powered only during movement, then turned off to prevent jitter.
+//#define DEACTIVATE_SERVOS_AFTER_MOVE
+
+
+// With probe what is triggered when stowed we can test if it is deployed/stowed or not.
+// This is common for Allen Key probes and some servo probes.
+//#define PROBE_IS_TRIGGERED_WHEN_STOWED_TEST
+
+// Allen key retractable z-probe as seen on many Kossel delta printers - http://reprap.org/wiki/Kossel#Automatic_bed_leveling_probe
+// Deploys by touching z-axis belt. Retracts by pushing the probe down. Uses Z_MIN_PIN.
+//#define Z_PROBE_ALLEN_KEY
+// Allow homing with fake Allen key probe. A debug feature.
+#define TEST_ALLEN_KEY
+
+#if ENABLED(Z_PROBE_ALLEN_KEY)
+  // 2 or 3 sets of coordinates for deploying and retracting the spring loaded touch probe on G29,
+  // if servo actuated touch probe is not defined. Uncomment as appropriate for your printer/probe.
+
+  // Cartesian: deploy far right, far back; stow with probe at X_MAX_POS,Y_MAX_POS
+  #define ALLEN_KEY_DEPLOY_WAY 30.0
+  #define Z_PROBE_ALLEN_KEY_DEPLOY_1_X (X_MAX_POS - ALLEN_KEY_DEPLOY_WAY)
+  #define Z_PROBE_ALLEN_KEY_DEPLOY_1_Y sqrt(sq((float)Y_MAX_POS) - sq((float)ALLEN_KEY_DEPLOY_WAY))
+  #define Z_PROBE_ALLEN_KEY_DEPLOY_1_Z (Z_MIN_POS + (Z_MAX_POS-Z_MIN_POS)/5)
```

Comment:
In fact, there are several potential errors here. The full correction needs some extra parentheses…
```cpp
#define Z_PROBE_ALLEN_KEY_DEPLOY_1_X (X_MAX_POS - (ALLEN_KEY_DEPLOY_WAY))
#define Z_PROBE_ALLEN_KEY_DEPLOY_1_Y sqrt(sq(float(Y_MAX_POS)) - sq(float(ALLEN_KEY_DEPLOY_WAY)))
#define Z_PROBE_ALLEN_KEY_DEPLOY_1_Z (Z_MIN_POS + (Z_MAX_POS-(Z_MIN_POS))/5)
```